cubi_tk/snappy/itransfer_common.py

```diff
@@ -380,15 +383,15 @@ def get_sodar_info(self):
                 # Behavior: search for available LZ; if none,create new LZ.
                 try:
                     lz_uuid, lz_irods_path = self.get_latest_landing_zone(
-                        project_uuid=in_destination
+                        project_uuid=in_destination, assay_uuid=self.args.assay
```

Comment:
Just to keep it consistent with the rest of the code I would suggest a copy of the variable in `# Initialise variables` - Just to make clear which variables are important for the method right from the jump.
Like: 
```
        not_project_uuid = False
        create_lz_bool = self.args.yes
        assay_uuid = self.args.assay  # <<< HERE
        in_destination = self.args.destination

```